microsoft-edge/devtools-guide-chromium/crash-analyzer/index.md

```diff
@@ -6,108 +6,108 @@ ms.author: msedgedevrel
 
+The **Crash analyzer** tool doesn't collect stack traces for you. You must first collect stack traces from your web app by using the tools and services that are available to you.  Here are some ways to collect stack traces in production:
 
-<!-- ====================================================================== -->
-## Store source maps using Azure Artifacts Symbol Server
+* We recommend using a telemetry system such as [Azure Monitor Application Insights](/azure/azure-monitor/app/app-insights-overview) which can capture unhandled error information from your JavaScript code in production.
 
-The **Crash analyzer** tool works best if your source maps are securely stored using Azure Artifacts Symbol Server.  This is because source maps are retrieved on demand when analyzing your error, and the exact version which produced the error message might not be cached on your development machine.
+* You can also write JavaScript code to capture unhandled errors in your web app.  The following example shows how to use the `window.onerror` event handler to capture unhandled errors in a web app:
 
-See [Securely debug original code by publishing source maps to the Azure Artifacts symbol server](../javascript/publish-source-maps-to-azure.md).
+  ```javascript
+  window.onerror = function (message, source, line, column, error) {
+    // Get the stack trace from the error object.
+    const stackTrace = error.stack;
 
+    // Send the stack trace to your telemetry system.
+    // Code not shown.
+  };
+  ```
 
-<!-- ====================================================================== -->
-## Use the Crash Analyzer Support npm package to collect stack traces
 
-Use the [Crash Analyzer Support](https://www.npmjs.com/package/@microsoft/edge-devtools-crash-analyzer-support) npm package in your web app.  The Crash Analyzer Support package produces stack traces in the format that's needed by the **Crash analyzer** tool.  To use the library that's in the Crash Analyzer Support npm package, you import the Crash Analyzer Support library, and then call `installErrorStackModuleAnnotations(Error);`.
+<!-- ------------------------------ -->
+#### Format the stack traces for the Crash analyzer tool
 
-Produce a specially-formatted stack trace that includes a section called `Source modules`.  The `Source modules` section contains the source map references to the JavaScript functions that were part of the error stack trace.  You can produce such a stack trace by using the [Crash Analyzer Support](https://www.npmjs.com/package/@microsoft/edge-devtools-crash-analyzer-support) npm package.
+Collecting stack traces in production is not enough. The **Crash analyzer** tool needs stack traces to include a section called **Source modules**.
 
-To install the Crash Analyzer Support npm package, at a command prompt, such as git bash, enter the install command that's shown at the above page:
+Here is an example of a stack trace that includes the **Source modules** section:
```

Comment:
per corp level style guide, i'll probably change to "Here's", for more casual tone - tho that's really a guideline more for end-user docs.
@captainbrosset - it strikes me that contractions, as a desirable style, is probably originating more from end-user docs like Office, than dev docs POV.